docs/contributing/working.md

```diff
@@ -72,19 +72,19 @@ e142fd1 Rebase with master
 ac8326d Fix ts-lint errors
 ```
 
-As a core philosophy we prefer [rebasing over merging](https://www.atlassian.com/git/tutorials/merging-vs-rebasing)
+As a core philosophy, we prefer [rebasing over merging](https://www.atlassian.com/git/tutorials/merging-vs-rebasing)
 
 ## Common commands
 
-All "packages" share common script names for consistency along with their own documentation guides.
+All "packages" share common script names for consistency along with their documentation guides.
 
 To clean all packages.
 
 ```bash
 lerna clean
 ```
 
-When package files change the symbolic links will require updating.
+When package files change the symbolic links require updating.
```

Comment:
This took me a second to understand because I thought we were talking about `package.json` files. Maybe we could rephrase to "build and install all package dependencies" or "install remote dependencies, and build and symlink all local dependencies"